.github/workflows/ci.yml

```diff
@@ -150,6 +155,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: docker/setup-buildx-action@v2
+        with:
+          driver-opts: network=host
```

Comment:
Allows `docker buildx` to talk to local registry by ensuring the buildx container is on the same network at the host.